src/app/public/modules/modal/modal-component-adapter.service.ts

```diff
@@ -97,13 +92,26 @@ export class SkyModalComponentAdapterService {
         inputWithAutofocus.focus();
       } else {
         let focusEl: HTMLElement = modalEl.nativeElement.querySelector('.sky-modal-content');
-        focusEl.focus();
+        let focussableChildren = this.loadFocussableChildren(focusEl);
 
+        // Focus first focussable child if available. Otherwise focus content pane.
+        if (!this.focusFirstElement(focussableChildren)) {
+          focusEl.focus();
+        }
       }
       window.scrollTo(currentScrollX, currentScrollY);
     }
   }
 
+  private loadFocussableChildren(elem: HTMLElement) {
```

Comment:
There's a few instances of the misspelled "focusable" (just one "s").

Reply:
Was wondering about that. Probably should have looked it up to confirm spelling ;p
Fixed now though. Z-index too